CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 After upgrading, update your cache file by deleting it or via `tldextract
 --update`.
 
+## 2.1.0 (2017-05-24)
```

Comment:
I'll try to rebase again before fixing the other issues; might an issue with this version of git. If so, I'll try from another machine.  Github keeps thinking the existing merge already in master is new.